src/content/en/updates/posts/2014/11/Support-for-installable-web-apps-with-webapp-manifest-in-chrome-38-for-Android.markdown

```diff
@@ -133,29 +133,29 @@ Taking a look at native apps, the majority of News based apps treat their experi
 
 A question that keeps popping up about Chrome's implementation of "mobile-web-app-capable" is: How can I tell if
 I am running as launched from the home screen?  On iOS you can use `navigator.standalone` to see if it is running
-like a native app, but we don't have this on Chrome for Android.
+like a native app, but we don't have this on Chrome for Android. [Instead](https://code.google.com/p/chromium/issues/detail?id=289113), you should [use the `display-mode`](https://developers.google.com/web/updates/2015/10/display-mode) media query feature.
+
 
-A naive approach is: Because you can now define how your app is launched, add a query string parameter to your `start_url` that indicates how it was launched.  For example `start_url: /index.html?homescreen=1`.  This will help in some cases but it is not infallible.
 
 # Learning More
 
-Check out Chrome's guidance for [Add to home screen](https://developer.chrome.com/multidevice/android/installtohomescreen#supporting) the [HTML5Doctor](http://html5doctor.com/web-manifest-specification/) for a deeper dive in
-to where the future of the spec and implementations will be
+Check out Chrome's guidance for [installing to the home screen](https://developer.chrome.com/multidevice/android/installtohomescreen#supporting) and the [HTML5Doctor](http://html5doctor.com/web-manifest-specification/) for a deeper dive in
```

Comment:
Will fix. I was misled by the URL of https://developer.chrome.com/multidevice/android/installtohomescreen (and the "homescreen-installed apps" terminology there).